docs/release-notes/pods-to-jobs.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes: When users submit Determined tasks, the system now launch Kubernetes jobs on their
+   behalf of instead of Kubernetes pods. The new system interacts differently with other Kubernetes
+   concepts; e.g., Determined tasks will now work properly with Kubernetes resources quotas.
+
+   Because of this, we now require permissions to create, get, list, delete and watch Kubernetes
+   jobs resources.
```

Comment:
:orphan:

**New Features**

- **Kubernetes**: The system now launches Kubernetes jobs on behalf of users when they submit Determined tasks, instead of launching Kubernetes pods. This change allows Determined tasks to work properly with Kubernetes resource quotas and interacts differently with other Kubernetes concepts.

   As a result, permissions are now required to create, get, list, delete, and watch Kubernetes job resources.

Reply:
what about

- The new system interacts differently with other Kubernetes concepts; e.g., Determined tasks will now work properly with Kubernetes resources quotas.
+ This change allows Determined tasks to work properly with other Kubernetes features like resource quotas.

Reply:
and maybe
The system now launches Kubernetes jobs on behalf of users when they submit workloads to Determined, instead of launching Kubernetes pods.

Reply:
so in its entirety:

:orphan:

**New Features**

- **Kubernetes**: The system now launches Kubernetes jobs on behalf of users when they submit workloads to Determined, instead of launching Kubernetes pods. This change allows Determined to work properly with other Kubernetes features like resource quotas.

   As a result, permissions are now required to create, get, list, delete, and watch Kubernetes job resources.